.github/workflows/ci-master.yml

```diff
@@ -55,4 +55,11 @@ jobs:
       with:
           name: CORONA_TRACING_BACKEND001_PRD-PROD
           path: /home/runner/work/efgs-federation-gateway/efgs-federation-gateway/target/CORONA_TRACING_BACKEND001_PRD*
-
+    - name: mvn clean install -P docker
+      run: mvn clean install -P docker
+    - name: Log into registry
+      run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u ${{ github.actor }} --password-stdin
```

Comment:
Do I understand the pipe right, that the echo will not print it to console but to the docker-command? Can you assure that the Github-Token will not be printed to CI logs?

Reply:
Yes it will pass the token to docker login and will not print to to console.
Example:
<img width="930" alt="Screenshot 2020-09-15 at 12 29 38" src="https://user-images.githubusercontent.com/8155/93193173-2e54ae00-f74f-11ea-9476-b7b5a71dd594.png">